README.md

```diff
@@ -67,7 +67,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
--   https://porkalaikarthika.github.io/
+-   https://github.com/Porkalaikarthika/Portfolio
```

Comment:
- [ ] The link you added to your readme file is not showing your deployed portfolio rather it takes me to your github repo as shown in the image below. please kindly fix this and resubmit :+1: :sparkles: 
- your link
![jc](https://github.com/Porkalaikarthika/Portfolio/assets/89055502/d4fb4361-e832-42f4-9991-99d6e7b8d4e3)